src/main/java/no/nav/fo/veilarbregistrering/kafka/KontaktBrukerOpprettetKafkaConsumer.java

```diff
@@ -0,0 +1,62 @@
+package no.nav.fo.veilarbregistrering.kafka;
+
+import no.nav.arbeid.soker.oppgave.KontaktBrukerOpprettetEvent;
+import no.nav.fo.veilarbregistrering.bruker.AktorId;
+import no.nav.fo.veilarbregistrering.registrering.bruker.DatakvalitetOppholdstillatelseService;
+import no.nav.sbl.featuretoggle.unleash.UnleashService;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.concurrent.Executors;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+/**
+ * 1. Den skal konsumere TOPIC for "Kontakt bruker opprettet"
+ * 2. Den skal kjøre i evig løkka
+ * 3. Den skal kalle på et internt API for å hente oppholdstillatelse
+ */
+public class KontaktBrukerOpprettetKafkaConsumer implements Runnable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(KontaktBrukerOpprettetKafkaConsumer.class);
+
+    private final KafkaConsumer<String, KontaktBrukerOpprettetEvent> consumer;
+    private final UnleashService unleashService;
+    private final DatakvalitetOppholdstillatelseService bruker;
+
+    public KontaktBrukerOpprettetKafkaConsumer(
+            KafkaConsumer<String, KontaktBrukerOpprettetEvent> consumer,
+            UnleashService unleashService,
+            DatakvalitetOppholdstillatelseService bruker) {
+        this.consumer = consumer;
+        this.unleashService = unleashService;
+        this.bruker = bruker;
+
+        Executors.newSingleThreadScheduledExecutor()
+                .schedule(this, 5, MINUTES);
+    }
+
+    @Override
+    public void run() {
+        LOG.info("Running");
+
+        while (konsumeringAvKontaktBruker()) {
+            ConsumerRecords<String, KontaktBrukerOpprettetEvent> consumerRecords = consumer.poll(Duration.ofMinutes(2));
```

Comment:
Har du noen tanker om hyppighet her? Jeg tenker at disse meldingene vi her lytter på ikke er veldig mange, 1000 ila en dag, og at vi derfor kan polle mindre ofte. At hvert 2 minutt kanskje er ok.

Reply:
Jeg ville nok fortsatt pollet med en duration ikke høyere enn en 5 sekunder. en poll er billig, du har en åpen connection, og da slipper du delayet på meldingene. Jeg pleier å kjøre med Duration.ofMillis(100), men det er nok hovedsaklig for jeg er vant til at det kommer nok meldinger. Duration.ofSeconds(1) er en typisk god loop